approval_polls/urls.py

```diff
@@ -6,6 +6,7 @@
     url(r'^$', views.index, name='index'),
     url(r'^(?P<pk>\d+)/$', views.DetailView.as_view(), name='detail'),
     url(r'^(?P<pk>\d+)/results/$', views.ResultsView.as_view(), name='results'),
+    url(r'^myPolls/$', views.PollsView, name='myPolls'),
```

Comment:
Better to use hyphen to separate words rather than camelcase (read more [here](http://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)). So the url will be 'my-polls'